### ISSUE TITLE
关于self.num_points和self.num_levels

### ISSUE THREAD

**Issue by kingofstu** [MCG-NJU/SparseBEV#73]
sampling_points = torch.cat([       # 将预测坐标恢复至每帧
            sampling_points[..., 0:2] - dist,
            sampling_points[..., 2:3]
        ], dim=-1)

        # scale weights
        scale_weights = self.scale_weights(query_feat).view(B, Q, self.num_groups, 1, self.num_points, self.num_levels)
        scale_weights = torch.softmax(scale_weights, dim=-1)
        scale_weights = scale_weights.expand(B, Q, self.num_groups, self.num_frames, self.num_points, self.num_levels)

您好，请问这个self.num_points和Bevformer中的self.num_points含义相同吗？还有self.num_levels和使用的FPN层数是对应的吗？

**Comment by afterthat97** [comment]
1. 差不多，都是采样点数量。
2. 对的

**Comment by kingofstu** [comment]
> 1. 差不多，都是采样点数量。
> 2. 对的

感谢回答
